uint8 local_index       # The number within the key chain
uint8 remote_addr       # The int dest addr (from ping payload) 
uint16 remote_index      # The int dest index (from ping payload)